File: Java/Card.java
package Java;

public class Card extends Payment {
    Integer numberCard;
    Integer CVVcode;
    String dateCard;

    public Card(Integer id, Integer numberCard, Integer CVVcode, String dateCard){
        super(id);
        this.numberCard = numberCard;
        this.CVVcode = CVVcode;
        this.dateCard = dateCard;
    }
}

File: Python/Card.py
from payment import Payment

class Card(Payment):
    numberCard = int
    CVVcode = int
    dateCard = str

    def __init__(self, id, numberCard, CVVcode, dateCard):
        super().__init__(id)
        self.numberCard = numberCard
        self.CVVcode = CVVcode
        self.dateCard = dateCard